docs/dev/new_check_howto.md

```diff
@@ -428,31 +460,57 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
 
 **Windows** (Ensure that your shell session has _administrator_ privileges):
 
-For Agent versions <= 6.11:
+<details>
+  <summary>Agent <code>v6.11</code> or earlier</summary>
+  
+  ```ps
+  "C:\Program Files\Datadog\Datadog Agent\embedded\agent.exe" integration install -w /path/to/wheel.whl
+  ```
 
-```ps
-"C:\Program Files\Datadog\Datadog Agent\embedded\agent.exe" integration install -w /path/to/wheel.whl
-```
+</details>
 
-For Agent versions >= 6.12:
+<details open>
+  <summary>Agent<code>v6.12</code> or later</summary>
 
-```ps
-"C:\Program Files\Datadog\Datadog Agent\bin\agent.exe" integration install -w /path/to/wheel.whl
-```
+  ```ps
+  "C:\Program Files\Datadog\Datadog Agent\bin\agent.exe" integration install -w /path/to/wheel.whl
+  ```
+</details>
```

Comment:
I think the Windows PowerShell commands need a leading `&` character:

  ```ps
  & "C:\Program Files\Datadog\Datadog Agent\embedded\agent.exe" integration install -w /path/to/wheel.whl
  ```
https://docs.datadoghq.com/agent/basic_agent_usage/windows/?tab=commandline#agent-commands

Also, let's indicate that the commands are for Windows PowerShell